8932 container base image (#8933)
IQSS/dataverse: doc/sphinx-guides/source/index.rst

  api/index
  installation/index
  developers/index
  container/index
  style/index

How the Guides Are Organized
----------------------------

The guides are documentation that explain how to use Dataverse,
which are divided into the following sections: User Guide,
Installation Guide, Developer Guide, API Guide, Style Guide and Container Guide.
The User Guide is further divided into primary activities: finding & using
data, adding Datasets, administering dataverses or Datasets, and Dataset exploration/visualizations. Details
on all of the above tasks can be found in the Users Guide. The
Installation Guide is for people or organizations who want to host their

> own Dataverse. The Container Guide adds to this information on container-based installations.

Reviewer: ```suggestion
own Dataverse. The Container Guide gives information on how to deploy Dataverse with containers.
```
